src/NeuralNetwork/index.js

```diff
@@ -556,7 +556,7 @@ class NeuralNetwork {
       results.tensor = ys;
     } else {
       results = {
-        value: await ys.data(),
+        values: await ys.data(),
```

Comment:
Ah yes this is also something to address what we call the outputs of the model. 
Just some ideas in addition to the above:
* values
* outputs
* data
* classifications/guesses
* estimations

I think `estimations` is interesting in that it relates to things like "pose estimation".

Reply:
I picked `value` originally as it matches the Feature Extractor, but then I realized (a) it's an array and (b) should be one b/c you can have more than one output. I guess in our image regression example we only allowed one output?

https://github.com/ml5js/ml5-examples/blob/release/p5js/FeatureExtractor/FeatureExtractor_Image_Regression/sketch.js#L86

I like `outputs` since it is very literal, I think `estimation` might not fit for all scenarios as it means something more specific? The simplest thing right now would be to go with `values` and update the image feature extractor regression to match with an array? Not sure at all!

Reply:
Ok, cool. Let's stick to `outputs` then. It definitely is the most literal!

Reply:
I will update the examples!